sdks/python/src/opik/url_helpers.py

```diff
@@ -35,6 +36,15 @@ def get_projects_url(workspace: str) -> str:
     return f"{ui_url}/{workspace}/projects/"
 
 
+def get_project_url(workspace: str, project_name: str) -> str:
+    ui_url = get_ui_url()
+
+    return urllib.parse.quote(
+        f"{ui_url}/{workspace}/redirect/projects?name={project_name}",
```

Comment:
Let's split it in 2 steps:

```python
project_path = urllib.parse.quote(f"{workspace}/redirect/projects?name={project_name}", safe=...)
return urllib.parse.join(ui_url, project_path)
```

It's safer to use join instead of strings formatting for concatenating url parts, because there are cases when someone adds/doesn't add `/` in the end or in the beginning of the url part. And it might lead to stuff like `https://www.opik.com//my_workspace`, join function is quite smart about such stuff.